pkg/api/config/config.go

```diff
@@ -19,13 +19,14 @@ var (
 )
 
 type StorageConfig struct {
-	RootDirectory string
-	GC            bool
-	Dedupe        bool
-	Commit        bool
-	GCDelay       time.Duration
-	GCInterval    time.Duration
-	StorageDriver map[string]interface{} `mapstructure:",omitempty"`
+	RootDirectory       string
+	GC                  bool
+	Dedupe              bool
+	Commit              bool
+	GCDelay             time.Duration
+	GCInterval          time.Duration
+	StorageDriver       map[string]interface{} `mapstructure:",omitempty"`
+	CacheDatabaseDriver map[string]interface{} `mapstructure:"cache,omitempty"`
```

Comment:
Just call it "CacheDriver", that it is a db is probably implementation detail

Reply:
Renamed to CacheDriver